borg/testsuite/archiver.py

```diff
@@ -933,7 +933,7 @@ def test_list_list_format(self):
         self.cmd('create', test_archive, src_dir)
         output_1 = self.cmd('list', test_archive)
         output_2 = self.cmd('list', '--list-format', '{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}', test_archive)
-        output_3 = self.cmd('list', '--list-format', '{mtime:%s} {path}{NL}', test_archive)
```

Comment:
Hmm, I just saw that 'NL' was not in the dict - but maybe it should be there...

The --format stuff is unrelated to my changes, but likely it could use same exception.